frontend/src/components/landing/Carousel.module.scss

```diff
@@ -139,7 +137,7 @@
     }
 
     &:focus .title {
-      border-color: hsla(0, 0%, 80%, 0.5);
+      box-shadow: none;
```

Comment:
There's still no focus state here, because you only restored the `border-color`, but not the actual border on `.title`, so the colour applies to nothing :)

Reply:
I added the `outline` property instead as the focus state.

Reply:
Woah, TIL about `outline`, neat!